api/rs/slint/Cargo.toml

```
@@ -18,6 +18,8 @@ rust-version.workspace = true
 path = "lib.rs"
 
 [features]
+serde = ["i-slint-core/serde"]
+serde-enabled = ["dep:serde", "serde"]
```

You do not need the serde enabled feature
```suggestion
serde = ["i-slint-core/serde", "dep:serde"]
```

But also, i don't think the slint crate needs to directly depend on serde, so this should either be moved as a dev-dependency, or removed completely.

```suggestion
serde = ["i-slint-core/serde"]
```

---

I suppose it should removed completely

---

@ogoffart there are some error on test on uefi-demo and mcu .
i can't recognize it . can you guide what are those ?

---

I suppose it will be great if we have optional for serde_json and use as a flag there